diff -up extra-cmake-modules-5.95.0/kde-modules/KDECompilerSettings.cmake.ldfw~ extra-cmake-modules-5.95.0/kde-modules/KDECompilerSettings.cmake
--- extra-cmake-modules-5.95.0/kde-modules/KDECompilerSettings.cmake.ldfw~	2022-06-13 11:08:55.647251991 +0200
+++ extra-cmake-modules-5.95.0/kde-modules/KDECompilerSettings.cmake	2022-06-13 11:10:41.619651079 +0200
@@ -534,8 +534,8 @@ if ((CMAKE_CXX_COMPILER_ID STREQUAL "GNU
         (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND NOT APPLE) OR
         (CMAKE_CXX_COMPILER_ID STREQUAL "Intel" AND NOT WIN32))
     # Linker warnings should be treated as errors
-    set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--fatal-warnings ${CMAKE_SHARED_LINKER_FLAGS}")
-    set(CMAKE_MODULE_LINKER_FLAGS "-Wl,--fatal-warnings ${CMAKE_MODULE_LINKER_FLAGS}")
+    #set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--fatal-warnings ${CMAKE_SHARED_LINKER_FLAGS}")
+    #set(CMAKE_MODULE_LINKER_FLAGS "-Wl,--fatal-warnings ${CMAKE_MODULE_LINKER_FLAGS}")
 
     # Do not allow undefined symbols, even in non-symbolic shared libraries
     # On OpenBSD we must disable this to allow the stuff to properly compile without explicit libc specification
