diff -up extra-cmake-modules-5.57.0/find-modules/FindPythonModuleGeneration.cmake.omv~ extra-cmake-modules-5.57.0/find-modules/FindPythonModuleGeneration.cmake
--- extra-cmake-modules-5.57.0/find-modules/FindPythonModuleGeneration.cmake.omv~	2019-04-11 21:58:58.345706289 +0200
+++ extra-cmake-modules-5.57.0/find-modules/FindPythonModuleGeneration.cmake	2019-04-11 21:59:04.960830424 +0200
@@ -249,7 +249,7 @@ if (NOT GBP_SIP_COMMAND)
 endif()
 
 if (NOT libclang_LIBRARY)
-  set(_LIBCLANG_MAX_MAJOR_VERSION 7)
+  set(_LIBCLANG_MAX_MAJOR_VERSION 10)
   set(_LIBCLANG_MIN_MAJOR_VERSION 5)
 
   find_library(libclang_LIBRARY clang-${_LIBCLANG_MAX_MAJOR_VERSION}.0)
